roottest/cling/templateMembers/constructor.C

```diff
@@ -19,7 +19,7 @@ struct A {
   template <class U>
    A(const U&) { 
       cout << "c'tor taking " << 
-#if defined(__CINT__) || defined(_MSC_VER)
+#if defined(__CLING__) || defined(_MSC_VER)
```

Comment:
This category of changes, I would keep the __CINT__ for now or actually review whether it is needed at all:
```suggestion
#if defined(_MSC_VER)
```